helmfile/charts/grafana-ops/dashboards/prometheus-blackbox-exporter-dashboard.json

```diff
@@ -0,0 +1,1213 @@
+{
+   "description" : "Prometheus Blackbox Exporter Overview",
+   "editable" : true,
+   "fiscalYearStartMonth" : 0,
+   "gnetId" : 7587,
+   "graphTooltip" : 0,
+   "id" : 60,
+   "links" : [],
+   "liveNow" : false,
+   "panels" : [
+      {
+         "datasource" : {
+            "type" : "prometheus",
+            "uid" : "$datasource"
+         },
+         "fieldConfig" : {
+            "defaults" : {
+               "color" : {
+                  "mode" : "thresholds"
+               },
+               "custom" : {
+                  "fillOpacity" : 70,
+                  "lineWidth" : 1
+               },
+               "mappings" : [
+                  {
+                     "options" : {
+                        "0" : {
+                           "color" : "red",
+                           "index" : 0,
+                           "text" : "DOWN"
+                        },
+                        "1" : {
+                           "color" : "green",
+                           "index" : 1,
+                           "text" : "UP"
+                        }
+                     },
+                     "type" : "value"
+                  }
+               ],
+               "thresholds" : {
+                  "mode" : "absolute",
+                  "steps" : [
+                     {
+                        "color" : "red",
+                        "value" : null
+                     },
+                     {
+                        "color" : "red",
+                        "value" : 0
+                     },
+                     {
+                        "color" : "green",
+                        "value" : 1
+                     }
+                  ]
+               }
+            },
+            "overrides" : []
+         },
+         "gridPos" : {
+            "h" : 8,
+            "w" : 24,
+            "x" : 0,
+            "y" : 0
+         },
+         "id" : 381,
+         "links" : [],
+         "options" : {
+            "colWidth" : 0.9,
+            "legend" : {
+               "displayMode" : "list",
+               "placement" : "bottom",
+               "showLegend" : false
+            },
+            "rowHeight" : 0.9,
+            "showValue" : "auto",
+            "tooltip" : {
+               "mode" : "single",
+               "sort" : "none"
+            }
+         },
+         "pluginVersion" : "9.2.4",
+         "targets" : [
+            {
+               "datasource" : {
+                  "uid" : "$datasource"
+               },
+               "editorMode" : "code",
+               "expr" : "probe_success{cluster=~\"$cluster\",instance=~\"$target\"}",
```

Comment:
More readable if the `target` label is used instead of `instance`. Any reason not to? The variable is already named `$target` :D